app/controllers/users.js

```diff
@@ -32,3 +33,34 @@ exports.createUser = (req, res, next) => {
     })
     .catch(next);
 };
+
+const generateToken = user => {
+  return new Promise((resolve, reject) => {
+    jwt.sign({ id: user.id, email: user.email }, process.env.JWT_KEY, (err, token) => {
```

Comment:
dont take environment variables directly from `process.env` , add them to the config index file instead